django/contrib/admin/options.py

```diff
@@ -1122,17 +1126,20 @@ def response_change(self, request, obj):
         pk_value = obj._get_pk_val()
         preserved_filters = self.get_preserved_filters(request)
 
-        msg_dict = {'name': force_text(opts.verbose_name), 'obj': force_text(obj)}
+        msg_dict = {
+            'name': force_text(opts.verbose_name),
+            'obj': force_text('<a href="{0}">{1}</a>').format(urlquote(request.path), escape(obj))
```

Comment:
Please including a trailing comma in the last item of a dictionary so if more items are added we don't need to modify this line again.